src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -215,6 +245,28 @@ public void analyzeCommits_emptyCommits_success() throws ParseException {
         Assert.assertEquals(expectedCommitResults, actualCommitResults);
     }
 
+    @Test
+    public void analyzeCommits_commitsWithBinaryFileContribution_success() throws ParseException {
+        Author author = new Author(JAMES_AUTHOR_NAME);
+        List<CommitResult> expectedCommitResults = new ArrayList<>();
+
+        // binary file contribution will have 0 contribution and won't be added to fileTypesAndContributionMap
+        expectedCommitResults.add(new CommitResult(author, "a00c51138cbf5ab7d14f52b52abb182c8a369169",
+                parseGitStrictIsoDate("2020-04-06T16:41:10+08:00"), "Add binary file",
+                "", null, 0, 0, new HashMap<>()));
```

Comment:
Same as the one above

```suggestion
        expectedCommitResults.add(new CommitResult(author, "a00c51138cbf5ab7d14f52b52abb182c8a369169",
                parseGitStrictIsoDate("2020-04-06T16:41:10+08:00"), "Add binary file",
                "", null, 0, 0, Collections.emptyMap()));
```